migrations/dimensions-config/380002bcbb26.py

```diff
@@ -0,0 +1,84 @@
+"""Migration script for dimensions.yaml namespace=daf_butler version=1.
+
+Revision ID: 380002bcbb26
+Revises: f3bcee34f344
+Create Date: 2022-05-13 14:38:35.806960
+
+"""
+from typing import Callable, Dict
+
+from alembic import context
+from lsst.daf.butler_migrate.butler_attributes import ButlerAttributes
+
+# revision identifiers, used by Alembic.
+revision = "380002bcbb26"
+down_revision = "f3bcee34f344"
+branch_labels = None
+depends_on = None
+
+# Change this when making a copy for different branch
+_NAMESPACE = "daf_butler"
+
+
+def upgrade() -> None:
+    """Upgrade from version 0 to version 1.
+
+    Version 1 adds few simple things to dimensions.json that do not modify
+    the schema:
+
+        - changes "version" value from 0 to 1
+        - adds "namespace" key to the configuration
+        - adds healpix to skypix
+
+    """
+
+    def update_config(config: Dict) -> Dict:
+
+        version = config.get("version")
+        if version is None:
+            raise ValueError(f"dimensions.json does not specify version: {config}")
+        if version != 0:
+            raise ValueError(f"unexpected version number: {version}")
+
+        config["version"] = 1
+        config["namespace"] = _NAMESPACE
```

Comment:
Namespace validation happens at a different level, when the tool maps manager class+version to alembic revision. We abuse namespace name as a manager class name for this manager, so that namespace name is hashed into alembic revision. This guarantees that different namespace name will never match this branch of migrations.

Reply:
And, thinking about it again, there could be a chance that someone modifies `dimensions.json` without using migration tool, maybe it makes sense to add this check anyways.

Reply:
RE-reading my code again - in fact there could not be a situation when namespace and version do not match what we expect here, there is an explicit consistency check for contents of `dimensions.json` vs alembic revisions. So I should actually drop the check for version number here too, it is not going to fail.